### PR TITLE
Add create whatsapp integration endpoint

### PR DESCRIPTION
**Pull request by maleblond**
https://zendesk.atlassian.net/jira/software/projects/SCBE/boards/1811?assignee=5ce63d95f593d10d74a73670&selectedIssue=SCBE-85

I followed this doc to generate the new collection: https://docs.google.com/document/d/1Jkrx44s33V-U1u-S5ooaRG66M3PPdT-_MpKOb8Mloro/edit?pli=1# . It led to a noisy diff (some API response examples have been re-ordered), so here's the relevant part of this PR:
https://github.com/smooch/smooch-postman-collection-public/pull/9/files#diff-4b081629b5ae173add1b20828fb6033bR6194

### CODE REVIEW

**Comment by maleblond** [comment]
The new json file needs to be published to postman, and I'll need to update the `Run in postman` link in the readme I think. I need someone who has access to the postman Smooch workspace to do it for me 😬  @scottanthonymorin could you publish it once I get an approval please and give me the new link? :pray:

File: Smooch.postman_collection.json [hunk 1]
```diff
@@ -6289,6 +6189,84 @@
...
+										"value": "Bearer {{JWT}}"
+									},
+									{
+										"key": "Content-Type",
+										"value": "application/json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n\t\"type\": \"whatsapp\",\n    \"deploymentId\": \"<whatsappDeploymentId>\"\n}"
```
Comment (fxlemire) [inline Smooch.postman_collection.json]: You'd be disappointed if I did not nitpick you a bit, so here it is:

```suggestion
									"raw": "{\n    \"type\": \"whatsapp\",\n    \"deploymentId\": \"<whatsappDeploymentId>\"\n}"
```

File: Smooch.postman_collection.json [hunk 2]
```diff
@@ -6289,6 +6189,84 @@
...
+											{
+												"key": "Content-Type",
+												"name": "Content-Type",
+												"value": "application/json",
+												"type": "text"
+											}
+										],
+										"body": {
+											"mode": "raw",
+											"raw": "{\n\t\"type\": \"whatsapp\",\n    \"deploymentId\": \"<whatsappDeploymentId>\"\n}",
```
Comment (fxlemire) [inline Smooch.postman_collection.json]: Une p'tite dernière:

```suggestion
											"raw": "{\n    \"type\": \"whatsapp\",\n    \"deploymentId\": \"<whatsappDeploymentId>\"\n}",
```

**Review by fxlemire — APPROVED** [review]

File: Smooch.postman_collection.json (hunk 1 above)
Comment (maleblond) [reply Smooch.postman_collection.json]: :+1: mon ptit nitpickeux

**Review by fxlemire — APPROVED** [review]

File: Smooch.postman_collection.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
 	"info": {
-		"_postman_id": "ff3c83a7-35b4-4846-9b31-a1dbe93444ae",
+		"_postman_id": "fd404f3a-7403-41b3-afa5-edf6e12b579a",
```
Comment (b0uh) [inline Smooch.postman_collection.json:3]: I'm not sure this is supposed to change 🤔

**Review by b0uh — APPROVED** [review]
I remember talking about updating the postman collection with Scott not long ago and it seems to be quite painful to do :/ I'm wondering it the doc you linked in the description is still up to date.

File: Smooch.postman_collection.json (hunk 3 above)
Comment (maleblond) [reply Smooch.postman_collection.json:3]: me neither :joy: 

J'ai demandé de l'aide à Scott, mais il est en rush présentement et vu que ce n'est pas urgent, j'imagine que ça va attendre quelques semaines :D

**Review by jugarrit — APPROVED** [review]

**Pull request merged**